database/CoreDatabase.java

```diff
@@ -145,6 +145,10 @@ protected CoreDatabase(CoreDatabaseManager databaseMgr, String name, Factory.Ses
         isOpen = new AtomicBoolean(false);
     }
 
+    protected StatisticsCorrector statsCorrectorFactory() {
```

Comment:
Statistics corrector factory method.

TODO: rename to `createStatisticsCorrector`